infrastructure/snyk_api/snyk_api.go

```diff
@@ -111,27 +111,24 @@ func (s *SnykApiClientImpl) FeatureFlagStatus(featureFlagType FeatureFlagType) (
 	method := "FeatureFlagStatus"
 	var response FFResponse
 	log.Debug().Str("method", method).Msgf("API: Getting %s", featureFlagType)
-	path := path.Join("/cli-config/feature-flags/", string(featureFlagType))
+	p := path.Join("/cli-config/feature-flags/", string(featureFlagType))
```

Comment:
suggestion: `path.Join` doesn't feel like the right method here, as the second part is not a slash-separated path, or if it were, we would want to URL encode it. I'd just use `fmt.Sprintf("/cli-config/feature-flags/%s",string(featureFlagType))`

Reply:
I read that `path.Join` knows how to deal thing different OS filesystem but in this case, it's not needed I guess. I understand your explanation and I'm using your suggestion :)